HOW/tech/applications/cell-joining-code/cell-joining-code-v1.md

```diff
@@ -0,0 +1,110 @@
+---
+HOW: /tech/applications/cell-joining-code
+Source: /social/process/how/how-v1
+Status: Draft
+Created: 2024-11-23
+Version: idx/1
+---
+
+## Title
+
+Cell Joining Code
+
+## Authors
+
+@mattyg <matt@buildyourweb.app>, @zippy <eric@holochain.org>, Claude AI 3.5 Sonnet
+
+## Abstract
+
+This HOW proposes a standardized URI format for cell joining codes that enables conductors to join specific networks. The format encapsulates all necessary information for network joining in a single, shareable URI string, excluding the actual DNA.  The format also allows app-specific data to be carried so that UIs can display arbitrary data as needed for good 
+
+## Motivation
+
+Joining a Cell network requires multiple pieces of information to be shared: DNA hash, network seed, membrane proof, and app-specific metadata. Several apps have run into the need to share this information as a single string and implemented their own joining code format.
+
+A standardized joining code format would:
+- Simplify the process for inviting someone into a network
+- Simplify the implementation of sending and receiving network invitations in apps
+- Facilitate services providing "always-on" nodes for a particular Cell
```

Comment:
This proposal is for joining codes for a single Cell -- not for an entire happ (multiple cells).